sirepo/template/srw.py

```diff
@@ -2528,6 +2579,8 @@ def _template_text(text):
 
 
 def _update_report_range(report, ar2d, x_range, y_range):
+    import numpy as np
```

Comment:
I don't think this is necessary as np is imported above.

Reply:
[5e2a07d](https://github.com/radiasoft/sirepo/pull/6186/commits/5e2a07dea579641f9b481f897f742ccf2a53bd0f)